Rename ControlPayload Class to Direct Control Message (DCM)

aglinxinyuan opened the pull request:
In this PR, we will rename ControlPayload to Direct Control Message (DCM) to align with the terminology used in our discussions and papers.

See more details at: https://www.vldb.org/pvldb/vol17/p3580-wang.pdf

shengquan-ni reviewed (changes requested):
Fix: ‘ControlPayload’ -> ‘DirectControlMessagePayload’. 
Suggest: ‘ControlElement’ -> ‘DCMElement’.

aglinxinyuan commented:
> Fix: ‘ControlPayload’ -> ‘DirectControlMessagePayload’. Suggest: ‘ControlElement’ -> ‘DCMElement’.

Renamed: ‘ControlPayload’ -> ‘DirectControlMessagePayload’.
Renamed: ‘ControlElement’ -> ‘DCMElement’.
Renamed: ‘EmbeddedControlMessageElement’ -> ‘ECMElement’.
